### DISCUSSION TITLE
Is there a way to start a twitch stream muted?

### DISCUSSION THREAD

**Discussion by superclarkk** [streamlink/streamlink#3783] [Q&A] [answered]
The twitch stream I want to open has music playing. Is there a way to mute it automatically? I'm using VLC as the player.

**Comment by bastimeyer** [comment, marked as answer]
- https://streamlink.github.io/cli.html#cmdoption-player-args
- https://wiki.videolan.org/Documentation:Advanced_Use_of_VLC/#Audio_options
- https://wiki.videolan.org/VLC_command-line_help/

```bash
streamlink -p vlc -a "--volume 0 --no-volume-save" URL STREAM
```

**Reply by superclarkk** [reply]
Brilliant - thank you. This worked perfectly...

```
streamlink --twitch-disable-ads --player-args "--noaudio" https://www.twitch.tv/thestream/ best
```
